src/appleseed/renderer/kernel/aov/aovaccumulator.h

```diff
@@ -37,14 +37,16 @@
 // appleseed.foundation headers.
 #include "foundation/core/concepts/noncopyable.h"
 #include "foundation/image/color.h"
-#include "foundation/platform/compiler.h"
 #include "foundation/utility/autoreleaseptr.h"
 
+// Standard headers.
+#include <cstddef>
+
 // Forward declarations.
-namespace renderer { class Camera; }
-namespace renderer { class ShadingComponents; }
-namespace renderer { class ShadingPoint; }
-namespace renderer { class ShadingResult; }
+namespace renderer  { class Camera; }
```

Comment:
Two spaces? I would be interested to know the standard using this. Is it personal taste?